test/data/stages/firewall/diff.json

```diff
@@ -1,8 +1,11 @@
 {
   "added_files": [
-    "/etc/firewalld/zones/public.xml",
-    "/etc/firewalld/zones/public.xml.old"
+    "/etc/firewalld/firewalld.conf.old",
+    "/etc/firewalld/zones/trusted.xml",
+    "/etc/firewalld/zones/trusted.xml.old"
   ],
   "deleted_files": [],
-  "differences": {}
+  "differences": {
+    "/etc/firewalld/firewalld.conf": {"mode": [41471, 33152]}
```

Comment:
Just a note, that the file is originally a symlink to `firewalld-standard.conf`, that's the reason for mode change.